tests/unit/core/agents/agent_components/test_agent_config.py

```
@@ -0,0 +1,223 @@
+    return session
+
+
+class TestCoerceRequestDelay:
+    def test_default_zero(self):
+        assert _coerce_request_delay(_make_session()) == 0.0
+
+    def test_valid_delay(self):
+        assert _coerce_request_delay(_make_session({"request_delay": 1.5})) == 1.5
+
+    def test_negative_raises(self):
+        with pytest.raises(ValueError, match="between 0.0 and 60.0"):
```

Ruff (0.14.14)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)